app-editors/vis/vis-0.8.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="A vi-like editor based on Plan 9's structural regular expressions"
+DESCRIPTION="Vis aims to be a modern, legacy-free, simple yet efficient editor, combining the strengths of both vi(m) and sam."
+HOMEPAGE="https://github.com/martanne/vis"
+COPYRIGHT="2014-2020 Marc André Tanner, et al."
+LICENSE="ISC"
+REVISION="1"
+SOURCE_URI="https://github.com/martanne/vis/releases/download/v$portVersion/vis-$portVersion.tar.gz"
+CHECKSUM_SHA256="61b10d40f15c4db2ce16e9acf291dbb762da4cbccf0cf2a80b28d9ac998a39bd"
+PATCHES="vis-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	vis$secondaryArchSuffix = $portVersion
+	cmd:vis = $portVersion
+	cmd:vis_clipboard = $portVersion
+	cmd:vis_complete = $portVersion
+	cmd:vis_digraph = $portVersion
+	cmd:vis_menu = $portVersion
+	cmd:vis_open = $portVersion
```

Comment:
Actually this is fine if commands are installed in $prefix/bin

Reply:
I thought $prefix/bin/secondaryArchSubdir was the default for runConfigure when building for a secondary arch?

Reply:
Sure, I meant "if" the runConfigure is run with --bindir=$prefix/bin, then $commandSuffix isn't needed because it's always empty.

Reply:
When --omit-dirs is used this should be done together with $commandBinDir IMHO

Reply:
Could binDir be redefined for x86, so that runConfigure use directly the correct bin directory?

Reply:
You mean in haikuports default values? Not trivially, because in some cases we do want the two sets of binaries (the gcc2 ones in bin and the gcc10 ones in bin/x86). However, maybe there are not that many things where that's the case and we should change the default, and special-case the recipes that need to do this instead?

Reply:
That should be part of haikuporter I guess, but it should detect either architecture if a library comes with some config script (cmd), we should be able to have that for both architectures on 32bit.

Reply:
it feels like this triggered a deeper discussion (which is fine of course) - but I don't know what to do now!